fiboa_cli/datasets/fieldscapes_corsica_2022.py

```diff
@@ -0,0 +1,139 @@
+# TEMPLATE FOR A FIBOA CONVERTER
```

Comment:
I'd propose to rename the file to `fs_fr_cor` for consistency.

Reply:
Same as below.